### PR TITLE
Add go v1.13 support

### PR DESCRIPTION
**Pull request by aya-taka**
Add golang version `1.13` support to `.travis.yaml`
And fix [TestByURLFailure](https://travis-ci.com/seqsense/aws-iot-device-sdk-go/jobs/262420824) because [net/url package updated](https://github.com/golang/go/commit/64cfe9fe22113cd6bc05a2c5d0cbe872b1b57860#diff-6c2d018290e298803c0c9419d8739885)

### CODE REVIEW

**Comment by aya-taka** [comment]
@at-wat @t-asaka please take a look

File: internal/awsiotprotocol/loader_test.go [hunk 1]
```diff
@@ -36,19 +37,23 @@ func TestByURL(t *testing.T) {
...
-			t.Errorf("awsiotprotocol.ByURL should fail with invalid input.\ninput: %#v", v.input)
-		} else if !strings.Contains(err.Error(), v.expectedErrorMessage) {
-			t.Errorf("awsiotprotocol.ByURL should fail with error message which contains the following:\ninput: %#v\nexpected error message: %#v\nactual error message: %#v", v.input, v.expectedErrorMessage, err.Error())
+			t.Errorf("awsiotprotocol.ByURL should fail with invalid input.\ninput: %#v", input)
+		} else if _, ok := err.(*url.Error); !ok {
+			t.Errorf("awsiotprotocol.ByURL should fail with url.Error type:\ninput: %#v\nactual error: %#v", input, err)
 		}
-	}
+	})
+	t.Run("Un supported protocol", func(t *testing.T) {
```
Comment (at-wat) [inline internal/awsiotprotocol/loader_test.go]:
```suggestion
	t.Run("Unsupported protocol", func(t *testing.T) {
```

File: internal/awsiotprotocol/loader_test.go [hunk 2]
```diff
@@ -36,19 +37,23 @@ func TestByURL(t *testing.T) {
...
-	testcase := []struct {
-		input                string
-		expectedErrorMessage string
-	}{
-		{input: "@@://@@@/@@.@@@", expectedErrorMessage: "parse @@://@@@/@@.@@@: "},
-		{input: "https://non-supported.protocol.com", expectedErrorMessage: "Protocol \"https\" is not supported"},
-	}
-	for _, v := range testcase {
-		_, err := ByURL(v.input)
+	t.Run("Cannot parse url", func(t *testing.T) {
```
Comment (at-wat) [inline internal/awsiotprotocol/loader_test.go]: `Invalid URL` may be better

**Comment by aya-taka** [comment]
@at-wat Thank you for your comments
I changed sub-test name to suggested name https://github.com/seqsense/aws-iot-device-sdk-go/pull/53/commits/5ca8f93c7521272ddc15b2560cecc2abea89c482

**Comment by aya-taka** [comment]
Thank you for your checking

**Pull request merged**